topics/dev/tutorials/tool-from-scratch/tutorial.md

```diff
@@ -523,20 +546,20 @@ of the requirement would change the value for `@TOOL_VERSION@` (and reset the `@
 
 A basic example of the xml macros would look like this
 
-```xml
-<macros>
-    <xml name="example_parameter">
-        <param name="ex_1" type="integer" label="Example help text"/>
-        <param name="ex_2" type="text" label="Example help text"/>
-    </xml>
-</macros>
-```
+> ```xml
+> </macros>
+>     <xml name="example_parameter">
+>         <param name="ex_1" type="integer" label="Example help text"/>
+>         <param name="ex_2" type="text" label="Example help text"/>
+>     </xml>
+> </macros>
+> ```
```

Comment:
what's the reason for this? it seems like an unncessary blockquote without any of the stylistic tags that the GTN uses that might make this blockquoting useful

Reply:
The linting was failing without the blockquote for the command section because of the "#"s, so I was trying to keep consistent visuals. I can remove it for the other sections, if you'd prefer

Reply:
hmm, then that's a bug with the linting that we should fix. I'll look at it